releases/release-1.22/release-notes/major-themes.md

```diff
@@ -0,0 +1,59 @@
+
+### Kubernetes Node system swap support
+
+Every system admin or kubernetes user has in the same boat regarding setting up and using Kubernetes, disabled swap space. Using the swap space has its benefits and if a Linux user enables it on machine running kublet is in a world of hurt. With the release of Kubernetes 1.22, alpha support is available for nodes running with swap memory. This change allows administrators to enable the Kubelet to control swap memory use on Linux nodes for container workloads in limited and unlimited configurations.
+
+### Kuberentes Release Cadence Change
+
+We all had to adapt to change in our life even more so in the past year. The Kubernetes Release team was also affected from the pandemic and listen it is user base regarding the number of release in a calendar year. From April 23, 2021 it was made official that Kubernetes release cycle is decrimented by one, from 4 to 3, a year. You can read more in the official blog post [Kuberentes Release Cadence Change: Here’s What You Need To Know]()
+
+### Enable seccomp by default
+
+A new alpha feature gate SeccompDefault has been added to the kubelet together with a corresponding command line flag `--seccomp-default` and kubelet configuration. If both are enabled, then the kubelet will use the RuntimeDefault seccomp profile rather than Unconfined if they're not specified in the Pod or Container SecurityContext. This allows enhancing the default cluster wide workload security of the Kubernetes deployment. Security administrators will now sleep better knowing there is some security by default for the workloads.
+To learn more about the feature, please refer to the [official documentation](https://kubernetes.io/docs/tutorials/clusters/seccomp/#enable-the-use-of-runtimedefault-as-the-default-seccomp-profile-for-all-workloads) and look out for the feature blog post.
+
+### Support memory qos with cgroups v2
+
+In traditional `cgroups v1` implementation in Kubernetes, pod qos class and qos manager can only limit cpu resources, such as `cpu_shares`. Kubernetes cgroup manager uses `memory.limit_in_bytes` in `cgroups v1` to limit the memory capacity and uses `oom_scores` to determine the order of killing the container processes when OOM occurs. This implementation has downfalls as `Guaranteed` memory can not be fully reserved, page cache is at risk of being recycled. And `Burstable` memory allocation overcommit ( request < limit ) could increase OOM possibility when kernel memory runs out.
```

Comment:
How about:
```suggestion
Originally, Kubernetes used the v1 cgroups API. With that design, the QoS class for a Pod only applied to CPU resources (such as `cpu_shares`). The Kubernetes cgroup manager uses `memory.limit_in_bytes` in v1 cgroups to limit the memory capacity for a container, and uses `oom_scores` to recommend an order for killing container processes if an out-of-memory event occurs. This implementation has shortcomings: for `Guaranteed` Pods, memory can not be fully reserved, and the page cache is at risk of being recycled. For `Burstable` Pods, overcommitting memory (setting `request` less than `limit` ) could increase the risk of a container being killed when the Linux kernel detects an out of memory condition.
```

Reply:
I would move this alpha feature to the end of major themes as it is an early feature and needs to be explicitly enabled via a feature gate.